DotNetHowToAutocomplete/DotNetHowToAutocomplete/Views/Home/IndexJavaScript.cshtml

```diff
@@ -0,0 +1,345 @@
+﻿@using System.Configuration;
```

Comment:
I recommend having someone like @jj09 review your HTML, JS, and CSS. It's all Greek to me. :)

Reply:
I wrote this HTML/JS/CSS :)

Reply:
Then I guess it's good :)